balboa-core/src/main/scala/com/socrata/balboa/metrics/data/impl/Cassandra11QueryImpl.scala

```diff
@@ -1,41 +1,39 @@
 package com.socrata.balboa.metrics.data.impl
 
 import java.io.IOException
+import java.nio.charset.Charset
 import java.{util => ju}
 
-import com.netflix.astyanax.connectionpool.OperationResult
-import com.netflix.astyanax.model.{ColumnList, ConsistencyLevel, Row}
-import com.netflix.astyanax.retry.ExponentialBackoff
-import com.netflix.astyanax.{AstyanaxContext, Keyspace, MutationBatch}
+import com.datastax.driver.core.querybuilder.QueryBuilder
+import com.datastax.driver.core.{BatchStatement, ConsistencyLevel, Row}
 import com.socrata.balboa.metrics.Metric.RecordType
+import com.socrata.balboa.metrics.data.impl.Cassandra11Util.DatastaxContext
 import com.socrata.balboa.metrics.data.{BalboaFastFailCheck, Period}
 import com.socrata.balboa.metrics.{Metric, Metrics}
 import com.typesafe.scalalogging.slf4j.StrictLogging
 
-import scala.collection.JavaConverters._
 import scala.{collection => sc}
+import scala.collection.JavaConversions.asScalaIterator
```

Comment:
Apparently there are problems with JavaConversions. `Conversions are perversions` is the matra. Prefer `JavaConverters`.